modules/data-entry/src/main/frontend/src/questionnaire/DateQuestionYear.jsx

```diff
@@ -52,7 +52,7 @@ function DateQuestionYear(props) {
   let {text, dateFormat, minAnswers, type, lowerLimit, upperLimit} = {dateFormat: "yyyy", minAnswers: 0, type: DateQuestionUtilities.TIMESTAMP_TYPE, ...props.questionDefinition, ...props};
   return (
     <NumberQuestion
-      minValue={0}
+      minAnswers={minAnswers || 0}
```

Comment:
`||0` is unnecessary, we already provide the `0` default value on line 52.
```suggestion
      minAnswers={minAnswers}
```